examples/mnist/batch_eth_mnist.py

```diff
@@ -9,6 +9,7 @@
 
 from time import time as t
 
+from bindsnet import ROOT_DIR
```

Comment:
I like this change! Didn't know we had that constant! 

Should we give some option as to the download path or no?

Reply:
I'd rather not add extraneous arguments. We have a lot already, and I like putting it under the project root directory as a default because it houses the data with the source code.